integrations/airflow/setup.cfg

```diff
@@ -11,6 +11,10 @@ replace = version="{new_version}"
 search = __version__ = "{current_version}"
 replace = __version__ = "{new_version}"
 
+[bumpversion:file:marquez_airflow/version.py]
```

Comment:
We can now remove the reference to `marquez_airflow/version.py` in `setup.cfg`

Reply:
Done